playwright/publisher.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { test, Page } from "@playwright/test";
+import {
+    checkIfOnDashboard,
+    checkReviewPage,
+    fillConsequenceJourneysPage,
+    fillConsequenceNetworkPage,
+    fillConsequenceOperatorPage,
+    fillConsequenceServicePage,
+    fillConsequenceStopsPage,
+    fillCreateDisruptionPage,
+    fillTypeOfConsequencePage,
+} from "./utils";
+
+test.use({ storageState: "playwright/.auth/publisher.json" });
+
+test("create network disruption", async ({ page }: { page: Page }) => {
```

Comment:
```suggestion
test("create network disruption", async ({ page }) => {
```
The `test` func is defined by the Playwright library so we don't need to explicitly declare types on the inner func's params. This can can apply to the tests further down as well.

Reply:
fixed